src/main/kotlin/net/irisshaders/lilybot/database/collections/ConfigCollection.kt

```diff
@@ -0,0 +1,140 @@
+@file:Suppress("DEPRECATION_ERROR")
+
+package net.irisshaders.lilybot.database.collections
+
+import com.kotlindiscord.kord.extensions.koin.KordExKoinComponent
+import dev.kord.common.entity.Snowflake
+import net.irisshaders.lilybot.database.Database
+import net.irisshaders.lilybot.database.entities.ConfigData
+import net.irisshaders.lilybot.database.entities.LoggingConfigData
+import net.irisshaders.lilybot.database.entities.ModerationConfigData
+import net.irisshaders.lilybot.database.entities.SupportConfigData
+import org.koin.core.component.inject
+import org.litote.kmongo.eq
+
+/**
+ * This object contains the functions or interacting with the [Logging Config Database][LoggingConfigData]. This object
+ * contains functions for getting, setting and removing logging config.
+ *
+ * @since 4.0.0
+ * @see getConfig
+ * @see setConfig
+ * @see clearConfig
+ */
+class LoggingConfigCollection : KordExKoinComponent {
+	private val configDb: Database by inject()
+
+	@PublishedApi
+	internal val collection = configDb.configDatabase.getCollection<LoggingConfigData>()
+
+	suspend inline fun getConfig(inputGuildId: Snowflake): LoggingConfigData? =
+		collection.findOne(LoggingConfigData::guildId eq inputGuildId)
+
+	suspend inline fun setConfig(loggingConfig: LoggingConfigData) {
+		collection.deleteOne(LoggingConfigData::guildId eq loggingConfig.guildId)
+		collection.insertOne(loggingConfig)
+	}
+
+	suspend inline fun clearConfig(inputGuildId: Snowflake) =
+		collection.deleteOne(LoggingConfigData::guildId eq inputGuildId)
+}
+
+/**
+ * This object contains the functions or interacting with the [Moderation Config Database][ModerationConfigData]. This
+ * object contains functions for getting, setting and removing logging config.
```

Comment:
Misleading docstring, I think?

Reply:
In need of updating. It was just thrown in there temporarily.